Documentation/ExtensionArchitecture/FileStructure/ExtTablesSql.rst

```diff
@@ -40,13 +46,123 @@ a database (because of the semi-complete table definitions allowed that
+
+..  _ext-tables-sql-types:
+
+Database types
+==============
+
+The following database types require special consideration if you use them:
+
+..  _ext-tables-sql-types-fixed-length:
+
+`CHAR` and `BINARY` as fixed length columns
+-------------------------------------------
+
+..  versionchanged:: 13.4
+    Fixed and variable length variants have been parsed already in the past,
+    but missed to flag the column as fixed for the fixed-length database field
+    types :sql:`CHAR` and :sql:`BINARY`. This resulted in the wrong creation of
+    these columns as :sql:`VARCHAR` and :sql:`VARBINARY`, which is now corrected.
+
+Not all database systems (RDBMS) act the same way for fixed-length columns.
+Implementation differences need to be respected to ensure the same query/data
+behaviour across all supported database systems.
+
+..  warning::
+    Using fixed-length :sql:`CHAR` and :sql:`BINARY` column types requires to carefully work
+    with data being persisted and retrieved from the database due to differently
+    behaviour specifically of PostgreSQL.
+
+..  tip::
+    :sql:`CHAR` and :sql:`BINARY` columns can be used (for storage or performance
+    adjustments), but only when composed data and queries take care of
+    database-system differences.
+
+    Otherwise, the "safe bet" is to consistently utilize :sql:`VARCHAR` and
+    :sql:`VARBINARY` columns types.
+
+..  _ext-tables-sql-types-char:
+
+Fixed-length SQL type `CHAR`
+----------------------------
+
+..  _ext-tables-sql-types-char-varchar-difference:
+
+Key Difference Between CHAR and VARCHAR
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The main difference between :sql:`CHAR` and :sql:`VARCHAR` is how the database
+stores character data in a database. :sql:`CHAR`, which stands for `character`,
+is a fixed-length data type, meaning it always reserves a specific amount of
+storage space for each value, regardless of whether the actual data occupies
+that space entirely. For example, if a column is defined as :sql:`CHAR(10)` and
+the word `apple` is stored inside of it, it will still occupy 10 characters worth of
+space (not just 5). Unused characters are padded with extra spaces.
+
+On the other hand, :sql:`VARCHAR`, short for `variable character`, is a
+variable-length data type. It only uses as much storage space as needed
+to store the actual data without padding. So, storing the word `apple` in a
+:sql:`VARCHAR(10)` column will only occupy 5 characters worth of
+space, leaving the remaining table row space available for other data.
+
+..  _ext-tables-sql-types-char-when:
+
+When to user `CHAR` columns
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Rule of thumb for fixed-length** :sql:`CHAR` **columns**
+
+*   Only use with **ensured fixed-length values** (so that no padding occurs).
+
+*   For 255 or more characters :sql:`VARCHAR` or :sql:`TEXT` must be used.
+
+..  _ext-tables-sql-types-char-hints:
+
+Hints on using fixed-length** `CHAR` columns
```

Comment:
```suggestion
**Hints on using fixed-length** `CHAR` columns
```

Reply:
Das ist ja schon eine Überschrift, die würde ich nicht noch zusätzlich fe

Reply:
I don't understand your 2 stars `**` there. I the header should not be bold, please remove the both stars.